interface/build.config.js

```diff
@@ -64,6 +64,12 @@ module.exports = {
    */
   vendor_files: {
     js: [
+      // For some unknown reason(s), jquery and jquery-ui library files MUST be
+      // put at the beginning of this array; Otherwise the ui-slider wouldn't
+      // show up on the web UI, and we got the following error in the console:
```

Comment:
Yes, they are dependencies for ui-slider, so I put them right before ui-slider library, but that is not enough. I tried a few locations for these two libraries, and seems that they have to be before angular.js too.